app/models/sprint.rb

```diff
@@ -74,6 +75,7 @@ class Sprint < Version
     where(['version_settings.project_id = ? AND version_settings.display = ?',
            project.id, VersionSetting::DISPLAY_RIGHT])
       .includes(:version_settings)
+      .references(:version_settings)
```

Comment:
the `references` is only necessary when you have string conditions AFAIK. You could thus rewrite the scope to:
```
where(version_settings: { project_id: project.id, display: VersionSettings::DISPLAY_RIGHT })
       .includes(:version_settings)
```